docs/getting-started-openstack.md

```diff
@@ -48,8 +48,7 @@ In this case, we want to use the Nova service:
 $compute = $client->computeService('nova', 'regionOne');
 ```
 
-The first argument is the __name__ of the service as it appears in the OpenStack service catalog. If in doubt, you can
-leave blank and it will revert to the default name for the service. The second argument is the region. The third and
+The first argument is the __name__ of the service as it appears in the OpenStack service catalog. For OpenStack users, this must be retrieved and entered in your code. The second argument is the region. The third and
```

Comment:
"must be retrived" --- do you think its worth linking this to a gist or something that shows how?